eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -77,7 +77,17 @@ case object ERR_INFORMATION_LEAK extends State
       8888888888     Y8P     8888888888 888    Y888     888     "Y8888P"
  */
 
-case class INPUT_INIT_FUNDER(temporaryChannelId: ByteVector32, fundingAmount: Satoshi, pushAmount: MilliSatoshi, initialFeeratePerKw: FeeratePerKw, fundingTxFeeratePerKw: FeeratePerKw, localParams: LocalParams, remote: ActorRef, remoteInit: Init, channelFlags: Byte, channelVersion: ChannelVersion)
+case class INPUT_INIT_FUNDER(temporaryChannelId: ByteVector32,
+                             fundingAmount: Satoshi,
+                             pushAmount: MilliSatoshi,
+                             initialFeeratePerKw: FeeratePerKw,
+                             fundingTxFeeratePerKw: FeeratePerKw,
+                             initialRelayFees_opt: Option[(MilliSatoshi, Int)],
```

Comment:
I suppose you have considered defining a like so?
```scala
case class RelayFees(base: MilliSatoshi, proportional: Int)
```

The only issue I would have is that I'm not sure how we should then define `ChannelUpdate`. Except if we define this `RelayFees` in the scope of the channel? In the future we will probably play a lot more with those feerates.

Reply:
Yes I did experiment with this, but I felt that then it made sense to use it (almost) everywhere in the codebase if we added this class, and it touched many files, so I figured we would do it in a separate PR to avoid hiding the real change.